docs/faq/events.md

```diff
@@ -28,3 +30,187 @@ There are lots of times when words just aren't enough to really get clarity on a
+
+```js
+el.dispatchEvent(
+  new Event('event', {
+    bubbles: true,
+  }),
+);
+```
+
+During the "bubble" phase an event travels element by element from the dispatching element up to the top of the current DOM tree. Often, `bubbles` is used to make your event available to ancestor elements, however, it isn't required to do so. During the "capture" phase, which occurs before the "bubble" phase the event will travel element by element down from the top of the current DOM tree, making the event available to those elements. However, listening to events on the "capture" phase can sometimes be confusing, involve extended techniques to add in declarative templating, and require extended documentation. In this way, the least work approach says using `bubbles` is a good idea and in cases where you do not, it is important to what you are committing to in choosing to make "capture" phase events a part of your element's API.
+
+If you'd like your event to be able to travel across shadow DOM boundaries, and begin its `capture` phase at the `window`, you'll need to manage the event's `composed` property, a la:
+
+```js
+el.dispatchEvent(
+  new Event('event', {
+    composed: true,
+  }),
+);
+```
+
+Be aware that a `composed` event will be retargeted to each shadow root that it passes through. In this way, the value for `event.target` will be the first of an element with a shadow DOM or the dispatching element as you travel from the listening element towards the dispatching element. This is one way in which a shadow root provides encapsulation for the inner working of your element. If you ever need the entire path across the DOM that your event will take you can use the `event.composedPath()` method to acquire an array of elements (starting with the dispatching element) that the event will pass through.
+
+With both of these properties managed, your event will both begin its `capture` phase at the `window` as well as a `bubble` phase which ends at the `window`. In this case, your event will look like the following:
+
+```js
+el.dispatchEvent(
+  new Event('event', {
+    bubbles: true,
+    composed: true,
+  }),
+);
+```
+
+Remember to be conscious of your use of `composed: true`. When using `composed`, all parent components and implementors will also have access to your event in both phases, which can lead to event pollution. Only use these settings when you mean for the entire application to have access to an event.
+
+## Even more data
+
+When you'd like to pass data along with your event, you have two options: dispatch a `new CustomEvent()` or extend the `Event` class. While no native DOM element uses the `CustomEvent` constructor to create the events that they dispatch, developers are offered this constructor and its additional `detail` property as a way to pass additional information along with their event. This can be done like so:
+
```

Comment:
add a "warning" that this can not be typed... so it would be better to use the approach below

Reply:
CustomEvent caan be typed with generics, why do people dislike it. Seems like creating a class is a lot more overhead

Reply:
I’m with Lars on this. What’s more very little of the Open WC content is directly in reference to TS, so I’m inclined to skip this. Let me know if you _really_ want it in.